engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -1558,6 +1558,10 @@ public void run()
           entry.getValue().recordingId = null;
         }
         for (ContainerStats.OperatorStats stats : statsList) {
+          if (stats == null) {
```

Comment:
Yes, I'll file another JIRA to correct CircularBuffer usage (single producer/single consumer contract) in the container.